src/ConversationTopic.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using StardewModdingAPI;
+using StardewValley;
+
+
+namespace QuestFramework
+{
+    class ConversationTopic
```

Comment:
Add accessor  `internal`
```suggestion
    internal class ConversationTopic
```

Reply:
Added to ConversationTopic